client/src/components/web-extension/RecoveryPhrase/RecoveryPhrase.scss

```diff
@@ -1,58 +1,59 @@
 .cd_we_recovery-keyphrase {
-    display: flex;
-    flex-flow: column nowrap;
-    justify-content: space-between;
-    height: 100%;
-    font-size: 16px;
-    padding: 20px 15px;
-
-    &__wrapper {
-        border-radius: 16px;
-        border: 1px solid var(--secondary-text-color-med);
-        display: flex;
-        flex-flow: column nowrap;
-        justify-content: center;
-        padding: 16px;
-    }
-
-    &__column {
-        display: flex;
-        flex-direction: column;
-        flex-wrap: wrap;
-        flex: 0 0 auto;
-        height: 280px;
-        justify-content: space-between;
-        list-style: none;
-        width: 50%;
-        padding: 0;
-    }
-
-    &__footer {
-        margin-top: 20px;
-        display: flex;
-        justify-content: center;
-    }
-
-    &__copy-btn {
-        width: 40%;
-    }
-    
-    &__box {
-        padding: 0 16px;
-    }
+	display: flex;
+	flex-flow: column nowrap;
+	justify-content: space-between;
+	height: 100%;
+	font-size: 16px;
+	padding: 20px 15px;
+
+	&__wrapper {
+		border-radius: 16px;
+		border: 1px solid var(--secondary-text-color-med);
+		display: flex;
+		flex-flow: column nowrap;
+		justify-content: center;
+		padding: 16px;
+	}
+
+	&__column {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+		flex: 0 0 auto;
+		height: 270px;
+		overflow: auto;
```

Comment:
`overflow: auto` hides the 2nd column on showing Recovery Phrase screen.
<img width="352" alt="image" src="https://user-images.githubusercontent.com/610234/198315267-e948fc76-0e91-4b90-aae9-2279160b888c.png">

Reply:
@duc-q Please help to review https://github.com/CasperDash/casperdash-client/pull/177 first, I fixed some UI issue there.